src/locales/en.json

```diff
@@ -0,0 +1,37 @@
+{
+	"player-status.choose": "Choose a radio to listen to",
```

Comment:
Nice locales refactoring and system of hierarchical labels "X.Y.Z"

Ideas: should we add on the to do list to add tests of locales consistency: is it translated in all languages? Are there deprecated keys? Is it a problem if a key is missing in some locale? Should we fallback to English when a field is not translated and have the tests hard fail if the key is missing in English?

Maybe one day there will be partial support for many languages with systems such as Transifex, the dev team won't know how to translate new fields in some languages.

Reply:
We definitely need some tests about locales consistency since translation helper would break on missing key. As you said, we should fallback to English and report an error.